common-rendering/src/components/Pagination.stories.tsx

```diff
@@ -1,13 +1,13 @@
-import { ArticleDesign, ArticleDisplay } from '@guardian/libs';
-import { breakpoints } from '@guardian/source-foundations';
-import { Pagination } from './Pagination';
-import { getAllThemes } from '../../../../common-rendering/src/fixtures/article';
+import { ArticleDesign, ArticleDisplay } from "@guardian/libs";
```

Comment:
why are all these quote marks changing? this kind of thing should never change in a PR (unless the PR is changeing the quote marks etc)

Reply:
our prettier config will use single quotes https://github.com/guardian/configs/blob/main/packages/prettier/index.js#L9